web_src/js/features/repo-diff.js

```diff
@@ -45,6 +45,9 @@ function initRepoDiffConversationForm() {
     e.preventDefault();
 
     const $form = $(e.target);
+    if ($form.hasClass('is-loading')) return;
+    $form.addClass('is-loading');
```

Comment:
Yes and no.
That class exists but is used for content that has not yet been loaded, not to click buttons only once.
There is a class that ensures buttons/forms are only submitted once.
I think it is `form-fetch-action` for forms or `link-action` for buttons/links.
Please use the correct class instead.

Reply:
The form-fetch-action uses this https://github.com/go-gitea/gitea/blob/main/web_src/js/features/common-global.js#L114-L116 and we cannot use form-fetch-action because the code comments has custom javascript code, which is initRepoDiffConversationForm.

Am I missing something?

Reply:
Okay, as @wxiaoguang pointed out in #28143, `is-loading` seems to be correct.
Sorry for that.
Anyway, #28143 does look better to me at the moment as it does not deadlock itself.